locations/spiders/south_carolina.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import re
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+class SCSpider(scrapy.Spider):
+    download_delay = 0.2
+    name = "south_carolina"
+    item_attributes = {'brand': "State of South Carolina", 'brand_wikidata': "Q53709977"}
```

Comment:
I'd leave these off since they don't really make sense as OSM brand tags. The `@spider` attribute is sufficient to identify what these items are.